docs/development/style-guide.rst

```diff
@@ -0,0 +1,237 @@
+* In some cases, most of the required functionality is contained in a single
+  class (or a few classes).
+
+**Quotation marks**: periods and commas will be placed inside of closing
+quotation marks, whether double or single.
+
+Examples:
+---------
+* Chief among these terms is the concept of a “coordinate system.”
+* Because of the likelihood of confusion between these meanings of “coordinate
+  system,” `~astropy.coordinates` avoids this term wherever possible.
+
+**Hyphens vs. En Dashes vs. Em Dashes**
+
+Hyphens (-) should be used for phrasal adjectives and compound words (see
+Hyphenation above).
+
+En dashes (–) should be used for number ranges (dates, times, pages) or to
+replace the words “to” or “through,” without spaces around the dash.
+Examples:
+* See chapters 14–18
+* We’ve blocked off March 2019–May 2019 to develop a new version.
+
+Em dashes (—) can be used in place of commas, parentheses, or colons to set off
+amplifying or explanatory elements. In Astropy materials, follow AP style, which
+calls for spaces on either side of each em dash.
+Examples:
+* Several types of input angles — array, scalar, tuple, string — can be used in
+  the creation of an Angle object.
+* The creation of an Angle object supports a variety of input angle types —
+  array, scalar, tuple, string, etc.
+
+Spelling
+========
+
+For consistency across Astropy materials, non-U.S. spelling will be edited to
+reflect American spelling preferences.
+
+Example:
+--------
+* Cross-matching catalog coordinates (versus catalogue)
+
+Time and Date
+=============
+
+Use numerals when exact times are expressed, followed by *ante meridiem* or
+*post meridiem* abbreviated in lowercase with periods, or use the twenty-four-
+hour system.
+
+Examples:
+---------
+* The meeting is set for 9:30 a.m.
+* The presentation starts at 15:00.
+
+Express specific dates as numerals in ISO 8601 format, year-month-day.
+
+Example:
+--------
+* Data from the Gaia mission was released on 2018-04-25.
+
+A Note About Style and Tone
+===========================
+
+Across all Astropy materials in narrative sections, please write in:
+
+* The present tense. For example:
+  * In the following section, we are going to make a plot…
+  * To test if your version of ``astropy`` is running correctly…
+
+* The first-person inclusive plural. For example:
+  * We did this the long way, but next let’s try the short way…
+
+* Use the generic pronoun “you” instead of “one.” For example:
+  * You can access any of the attributes on a frame by...
+
+Always avoid extraneous or belittling words such as “obviously,” “easily,”
+“simply,” “just,” or “straightforward.” Avoid extraneous phrases like, “we just
+have to do one more thing.”
+
+Documentation vs. Tutorials vs. Guides
+--------------------------------------
+
+Documentation
+~~~~~~~~~~~~~
+Tone: academic and slightly more formal.
+* Use up style capitalization in section headings.
+* Do not use contractions.
+
+Tutorials
```

Comment:
@kelle , @adrn , and @eblur , you might wanna cross-link tutorial doc to here, which also means this section might need a RST linking name thingy. FYI.

Reply:
Which tutorial doc?

Reply:
https://github.com/astropy/astropy-tutorials/tree/master/tutorials ?